sdk/tests/deqp/framework/referencerenderer/rrVertexAttrib.js

```diff
@@ -220,6 +220,19 @@ var rrGenericVector = framework.referencerenderer.rrGenericVector;
         DE_ASSERT(rrVertexAttrib.isValidVertexAttrib(vertexAttrib));
         /** @type {goog.NumberArray} */ var dst;
 
+        var arrayType = null;
```

Comment:
The C++ code is different from the original, and modified version and use templating.